napari/_qt/widgets/qt_progress_bar.py

```diff
@@ -50,8 +50,8 @@ def __init__(
 
         self.setLayout(base_layout)
 
-    @rename_argument("min", "min_val", "0.6.0")
-    @rename_argument("max", "max_val", "0.6.0")
+    @rename_argument("min", "min_val", "0.6.0", "0.4.18")
+    @rename_argument("max", "max_val", "0.6.0", "0.4.18")
```

Comment:
Personally I would prefer if these were used as keyword arguments, but I won't kick up a fuss about it 😅

Reply:
better now?